charts/nri-metadata-injection/README.md

```diff
@@ -13,11 +13,11 @@ This chart will deploy the [New Relic Infrastructure metadata injection webhook]
-| `image.tag`                     | The version of the container to pull.                                                                                                                                                                                                                                                                                                     | `1.4.0`                             |
+| `image.tag`                     | The version of the container to pull.                                                                                                                                                                                                                                                                                                     | `1.7.0`                             |
 | `jobImage.repository`           | The job container to pull.                                                                                                                                                                                                                                                                                                                | `newrelic/k8s-webhook-cert-manager` |
 | `jobImage.pullPolicy`           | The job pull policy.                                                                                                                                                                                                                                                                                                                      | `IfNotPresent`                      |
 | `jobImage.pullSecrets`          | The job pull secrets.                                                                                                                                                                                                                                                                                                                     | `nil`                               |
-| `jobImage.tag`                  | The job version of the container to pull.                                                                                                                                                                                                                                                                                                 | `1.4.0`                             |
+| `jobImage.tag`                  | The job version of the container to pull.                                                                                                                                                                                                                                                                                                 | `v1.1.1`                             |
```

Comment:
Are we downgrading? I know the answer is no because there are no change in the `values.yaml`.

But, do we have to upgrade the CA injector of something? 

Maybe I am missing something.

Reply:
I was fixing the readme, I guess at some point we edited the jobImage.tag with the nri-metadata tag

Reply:
this is what we have in the values:
```
jobImage:
  repository: k8s.gcr.io/ingress-nginx/kube-webhook-certgen
  tag: v1.1.1
  pullPolicy: IfNotPresent
```

Reply:
Yeah. But we have a v1.4.0 here: https://github.com/newrelic/k8s-webhook-cert-manager/releases

So we change the README and dis not update the values.

Would you upgrade the image to the last version?

Reply:
ahh ok, now I get it, no that is a deprecated repo, the image we use is now different, it is "k8s.gcr.io/ingress-nginx/kube-webhook-certgen"

Reply:
it used to be build from [here ](https://github.com/jet/kube-webhook-certgen), but now it has been incorporated in a k8s repo I am not able to find 😠

Reply:
https://github.com/kubernetes/ingress-nginx/tree/main/images/kube-webhook-certgen

Reply:
Makes sense. I was just not sure if we were failing `.jobImage.repository` or `jobImage.tag`.

Thanks!